python/BioSimSpace/Sandpit/Exscientia/FreeEnergy/_restraint.py

```diff
@@ -444,14 +457,24 @@ def write_angle(key_list, equilibrium_values, force_constants):
             return master_string.format(
                 index=format_index(key_list),
                 func_type=1,
-                parameters=format_angle(equilibrium_values, force_constants),
+                parameters=format_angle(
+                    equilibrium_values, force_constants, restraint_lambda=False
+                ),
             )
 
-        def write_dihedral(key_list, equilibrium_values, force_constants):
+        def write_dihedral(
+            key_list, equilibrium_values, force_constants, restraint_lambda
+        ):
+            if restraint_lambda:
+                func_type = 1
```

Comment:
Probably good to put some comments on what these mean or at least a link because I don't remember them off the top of my head